src/main/java/org/webonise/gitmetrics/controllers/webhooks/BranchController.java

```diff
@@ -0,0 +1,30 @@
+package org.webonise.gitmetrics.controllers.webhooks;
+
+import org.apache.catalina.servlet4preview.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.webonise.gitmetrics.services.interfaces.webhooks.BranchService;
+
+@Controller
+@RequestMapping("/webhooks/repository/branch")
+public class BranchController {
+    @Autowired
+    private BranchService service;
```

Comment:
name the variable to `branchService`.Makes it easy when a controller has to call many services